repo.json

```diff
@@ -130,7 +130,7 @@
                 "Atom_Feature_Common",
                 "Atom_Component_DebugCamera",
                 "CommonFeaturesAtom",
-                "PhysX",
+                "PhysX5",
```

Comment:
changing the gem data manually will invalidate the hash for the remote object.  If you use the o3de cli to update the repo.json it should update the hash as well, also we should bump the gem version so this change actually appears in the `versions_data` section that way we won't break older versions of the gem.